src/js/selection.js

```diff
@@ -77,14 +77,14 @@ var Selection;
 
         // http://stackoverflow.com/questions/4176923/html-of-selected-text
         // by Tim Down
-        getSelectionHtml: function getSelectionHtml() {
+        getSelectionHtml: function getSelectionHtml(doc) {
```

Comment:
If someone else use `getSelectionHtml` before your change, since the signature of the function has changed, it will breaks.
Could you fallback to `this.options.contentWindow` in case of `doc` is `undefined`?